libs/dal/src/repositories/base-repository.ts

```diff
@@ -105,6 +104,10 @@ export class BaseRepository<T> {
     };
   }
 
+  async bulkWrite(bulkOperations: any) {
+    await this.MongooseModel.bulkWrite(bulkOperations);
```

Comment:
Needed to update multiple organisations with different values, so chose to have bulkwrite method